scipy/optimize/_minpack_py.py

```diff
@@ -790,6 +790,11 @@ def curve_fit(f, xdata, ydata, p0=None, sigma=None, absolute_sigma=False,
     Box constraints can be handled by methods 'trf' and 'dogbox'. Refer to
     the docstring of `least_squares` for more information.
 
+    Parameters to be fitted should have similar scale. Differences of multiple
```

Comment:
According to the [documentation guidelines](https://docs.scipy.org/doc/scipy/dev/contributor/rendering_documentation.html#documentation-guidelines) you must use  `must` instead of `should` .

Reply:
Thanks, I missed that.